package.json

```diff
@@ -36,12 +36,13 @@
     "grunt-simple-mocha": "~0.4.0"
   },
   "dependencies": {
-    "commander": "1.1.1",
     "async": "0.1.22",
-    "jsdom": "^3.0.0",
-    "handlebars": "~1.0.12",
+    "can-simple-window": "git://github.com/canjs/can-simple-window",
```

Comment:
I [ashamedly] have never published to npm before. I would love to make that change. It would be sweet to get this out today as a user in gitter is waiting on it. Let me know what I can do.